proto/v1beta1/vulnerability.proto

```diff
@@ -149,7 +149,11 @@ message Vulnerability {
   // The full description of the CVSS for version 2.
   CVSS cvss_v2 = 7;
 
-  // Next free ID is 8.
+  // A list of CWE for this vulnerability.
+  // For details, see: https://cwe.mitre.org/index.html
+  repeated string cwe = 8;
```

Comment:
do we expect the same CWEs to apply on a per-distro basis? or will this vary?

Reply:
same CWEs should be for all distros.

Reply:
On checking further, CWEs are just common weakness types, they are general. CVEs are specific vulnerabilities that can affect one or more distros. We could keep CWE as a general field in the vulnerability message as in the PR.

Reply:
Got it, thanks for checking! SGTM. I checked a couple CVEs as well and it appears the CWE association is the same across distros.